app/src/main/java/com/realityexpander/tasky/core/presentation/broadcastReceivers/AlarmBroadcastReceiver.kt

```diff
@@ -19,6 +22,6 @@ class AlarmBroadcastReceiver : BroadcastReceiver() {
             return
         }
 
-        RemindAtNotificationManagerImpl.showNotification(context, alarmIntent)
+        remindAtNotificationManager.showNotification(alarmIntent)
```

Comment:
See here?

You forgot to create the notification channel (mentioned on another comment about the interface)